apps/teams-test-app/src/components/utils/ApiWithTextInput.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+
+import { noHostSdkMsg } from '../../App';
+import { ApiContainer } from './ApiContainer';
+
+export interface ApiWithTextInputProps<T> {
+  title: string;
+  name: string; // system identifiable unique name in context of Teams Client and should contain no spaces
```

Comment:
Can we validate that it has no spaces up here for everyone?

Reply:
hmm, I can certainly add runtime validation. It would be great to have that validated at build time though. Let me see if that's possible